schema-registry/common/src/test/java/com/hortonworks/registries/schemaregistry/util/CustomParameterizedBlockJUnit4Runner.java

```diff
@@ -0,0 +1,454 @@
+/**     JUnit
+
+ Eclipse Public License - v 1.0
```

Comment:
Shall we use the `junit-jupiter-params` library which provides inbuilt support to custom parameters?

Reply:
Yeah, we should. Currently, SR uses JUnit4, we need to upgrade it to JUnit5 to use custom parameters. I will create a Github issue for this.